### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#4] [Announcements]
# index

一份通俗易懂、风趣幽默的Java学习指南，内容涵盖Java基础、Java并发编程、Java虚拟机、Java企业级开发、Java面试等核心知识点。学Java，就认准Java程序员进阶之路

https://tobebetterjavaer.com/

**Comment by rao548** [comment]
星球专属的是微信群吗？

**Reply by itwanger** [reply]
不是微信群，是一个专属的私密学习交流圈子，你可以扫码体验一下，先感受一下星球的内容，目前已经沉淀了不少优质硬核的内容，觉得确实有帮助的话，可以直接上车。
